pkg/envoy/server.go

```diff
@@ -114,6 +114,9 @@ type XDSServer struct {
 	// Value holds the number of redirects using the listener named by the key.
 	listeners map[string]*Listener
 
+	// proxyListeners is the count of redirection proxy listeners in 'listeners'
```

Comment:
Maybe some additional context for the comment. E.g.,:

```suggestion
	// proxyListeners is the count of redirection proxy listeners in 'listeners'
	// When this is zero, cilium should not wait for NACKs/ACKs from envoy.
	// This value is different from len(listeners) due to non-proxy listeners (e.g., prometheus listener)
```

Reply:
applied.